packages/config/config/devices/0x0371/zwa009.json

```
@@ -17,39 +17,37 @@
 		"min": "0.0",
 		"max": "255.255"
 	},
-	"supportsZWavePlus": true,
 	"associations": {
 		"1": {
 			"label": "Lifeline",
 			"maxNodes": 5,
 			"isLifeline": true
 		},
 		"2": {
-			"label": "High Temperature Notification",
+			"label": "High Temperature",
 			"maxNodes": 5
 		},
 		"3": {
-			"label": "Low Temperature Notification",
+			"label": "Low Temperature",
 			"maxNodes": 5
 		},
 		"4": {
-			"label": "High Humidity Notification",
+			"label": "High Humidity",
 			"maxNodes": 5
 		},
 		"5": {
-			"label": "Low Humidity Notification",
+			"label": "Low Humidity",
 			"maxNodes": 5
 		},
 		"6": {
-			"label": "Air Temperature",
-			"description": "A multilevel sensor report is sent to the nodes in this group",
+			"label": "Air Temperature (MultiLevel)",
 			"maxNodes": 5
 		}
 	},
 	"paramInformation": {
 		"1": {
-			"label": "Minimum Temperature Change to Report",
-			"unit": "1/10 degree",
+			"label": "Temperature Change Report Threshhold",
```

```suggestion
			"label": "Temperature Change Report Threshold",
```
or am I mistaken here? Same happens in a few more location.